src/prg-6-7.asm

```diff
@@ -1217,18 +1223,30 @@ GreenPlatformOverwriteTiles:
 	.db BackgroundTile_Waterfall
 
 
+IFNDEF ENABLE_LEVEL_OBJECT_MODE
 ; Draw green platforms or mushroom house depending on world
```

Comment:
```suggestion
; Either draw typical green hill platforms (W1-6) or a mushroom-like platform (W7)
```

You technically didn't edit this line at all but I figure we might as well update the comment here too